SystemInformer/include/procprv.h

```diff
@@ -181,6 +181,7 @@ typedef struct _PH_PROCESS_ITEM
         ULONG Flags;
         struct
         {
+            ULONG IsTokenAccessible : 1;
```

Comment:
🤔 `TOKEN_ELEVATION_TYPE` starts at 1 and has been this way for eternity. Could this be used instead of consuming a flag?

```cpp
typedef enum _TOKEN_ELEVATION_TYPE {
    TokenElevationTypeDefault = 1,
    TokenElevationTypeFull,
    TokenElevationTypeLimited,
} TOKEN_ELEVATION_TYPE, *PTOKEN_ELEVATION_TYPE;
```

Reply:
Yes, now i'm using this :)